README.md

```diff
@@ -50,6 +50,12 @@ To install NML you'll need these 3rd party packages:
 - gcc (or possibly another c++ compiler).
   Needed to compile the cython version of the lz77 module for grf encoding.
 
+To improve NML and submit pull requests you'll need these code formaters and checkers.
```

Comment:
[pedant] "formatters".

Also this seems a bit vague. Non-contributors who want to run `make test` (e.g. distro packagers) will also need these, and they aren't strictly *required* for making a PR.

 Perhaps something like:

> Running `make test` requires these code formatters and checkers. You don't need these to use NML, only to test or help improve it.